lib/inquisition/collector.rb

```diff
@@ -6,12 +6,19 @@ def self.call
 
     def initialize
       @collection = Runner.collection
+      @config = Configuration.instance.to_h
     end
 
     def call
       @collection.each_with_object([]) do |runner, memo|
-        memo << runner.call
+        memo << runner.call if runner_enabled?(runner)
       end.flatten
     end
+
+    private
+
+    def runner_enabled?(runner)
```

Comment:
Maybe would be better if it will a state of the runner like an example `runner.enabled?`, what do you think?